src/Storage/Controllers/CleanupController.cs

```diff
@@ -18,45 +18,26 @@ namespace Altinn.Platform.Storage.Controllers
     /// </summary>
+    /// <remarks>
+    /// Initializes a new instance of the <see cref="CleanupController"/> class
+    /// </remarks>
+    /// <param name="instanceRepository">the instance repository handler</param>
+    /// <param name="applicationRepository">the application repository handler</param>
+    /// <param name="blobRepository">the blob repository handler</param>
+    /// <param name="dataRepository">the data repository handler</param>
+    /// <param name="instanceEventRepository">the instance event repository handler</param>
+    /// <param name="logger">the logger</param>
     [Route("storage/api/v1/cleanup")]
     [ApiController]
-    public class CleanupController : ControllerBase
+    public class CleanupController(
+        IInstanceRepository instanceRepository,
+        IApplicationRepository applicationRepository,
+        IBlobRepository blobRepository,
+        IDataRepository dataRepository,
+        IInstanceEventRepository instanceEventRepository,
+        ILogger<CleanupController> logger) : ControllerBase
     {
-        private readonly IInstanceRepository _instanceRepository;
-        private readonly IApplicationRepository _applicationRepository;
-        private readonly IBlobRepository _blobRepository;
-        private readonly IDataRepository _dataRepository;
-        private readonly IInstanceEventRepository _instanceEventRepository;
-        private readonly ILogger<CleanupController> _logger;
-        private readonly bool _usePostgreSQL;
-
-        /// <summary>
-        /// Initializes a new instance of the <see cref="CleanupController"/> class
-        /// </summary>
-        /// <param name="instanceRepository">the instance repository handler</param>
-        /// <param name="applicationRepository">the application repository handler</param>
-        /// <param name="blobRepository">the blob repository handler</param>
-        /// <param name="dataRepository">the data repository handler</param>
-        /// <param name="instanceEventRepository">the instance event repository handler</param>
-        /// <param name="logger">the logger</param>
-        /// <param name="generalSettings">the general settings.</param>
-        public CleanupController(
-            IInstanceRepository instanceRepository,
-            IApplicationRepository applicationRepository,
-            IBlobRepository blobRepository,
-            IDataRepository dataRepository,
-            IInstanceEventRepository instanceEventRepository,
-            ILogger<CleanupController> logger,
-            IOptions<GeneralSettings> generalSettings)
-        {
-            _instanceRepository = instanceRepository;
-            _applicationRepository = applicationRepository;
-            _blobRepository = blobRepository;
-            _dataRepository = dataRepository;
-            _instanceEventRepository = instanceEventRepository;
-            _logger = logger;
-            _usePostgreSQL = generalSettings.Value.UsePostgreSQL;
```

Comment:
There are unused usings after the cleanup. No need for Configuration for example.

Reply:
Ok